client/app/bundles/course/assessment/pages/AssessmentStatistics/StudentAttemptCountTable.tsx

```diff
@@ -22,20 +26,54 @@ import { getAssessmentStatistics } from './selectors';
 import translations from './translations';
 import { getJointGroupsName, translateStatus } from './utils';
 
+interface StudentSubmissionChipProps {
+  className?: string;
+  linkTo?: string;
+  workflowState?: WorkflowState | typeof workflowStates.Unstarted;
+}
+
+const StudentSubmissionChip: FC<StudentSubmissionChipProps> = (props) => {
+  const { className, linkTo, workflowState } = props;
+  const chip = (
+    <Chip
```

Comment:
Consider instead using `component={Link}` and `to={linkTo}` attributes instead.  
https://mui.com/material-ui/api/chip/#props